rcsbsearchapi/search.py

```diff
@@ -1224,6 +1240,161 @@ def __ge__(self, attr: Attr) -> Terminal:
         return attr.less_or_equal(self.value)
 
 
+@dataclass(frozen=True)
+class Range:
+    """Primarily for use with "range" and "date_range" aggregations with the Facet class.
+    include_upper and include_lower should not be used with Facet queries."""
```

Comment:
I'm trying to remember, is this _only_ for use with Facets? Or can it be used elsewhere too?

Reply:
Currently this is being used only with Facets, but a "range" can be specified for other types of queries too, which is why the include_lower and include_upper parameters exist too.